### PR TITLE
feat: [RAD-614] Ansible Molecule to test terraform role 

### PR DESCRIPTION
**Pull request by gamechanger1s**
Created a anisble role for installing terraform and tested with Ansible Molecule.

### CODE REVIEW

**Comment by maddy023** [comment]
@Jchinmay654  I have updated the title, please keep this as template for future PRs.
Also i have sent you other templates for fix and chore and make sure to fix the conflicts also 


Update the description to give reviewer a one or two liner about what exactly has been done.
Example: 
Created a terraform anisble role and tested with Ansible Molecule.

**Comment by gamechanger1s** [comment]
@maddy023  Understood, thank you for the information. I will keep this as a template for future pull requests.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+on:
+ workflow_dispatch: 
+  push:
+    branches:
+      - githubaction
+
+jobs:
+  molecule_test:
+    runs-on: ubuntu-latest
```
Comment (tarpanpathak) [inline .github/workflows/ci.yml]: Would it be possible to use an Ansible image here?

File: ansible-role/makefile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Run Molecule tests
+test:
+	molecule test
+
+# Build and test the role
+build: 
+	molecule converge
```
Comment (tarpanpathak) [inline ansible-role/makefile]: Maintain parity b/w the Make target and the command. E.g.:
```
converge:
    molecule converge

# OR

build:
    molecule build
```

File: .github/workflows/ci.yml (hunk 1 above)
Comment (gamechanger1s) [reply .github/workflows/ci.yml]: No it is not possible.

File: ansible-role/makefile (hunk 2 above)
Comment (gamechanger1s) [reply ansible-role/makefile]: Done

**Pull request closed without merging**